csrankings-0.csv

```diff
@@ -2188,8 +2188,8 @@ Benjamin Swift,Australian National University,https://cs.anu.edu.au/people/ben-s
 Benjamin Ujcich,Georgetown University,https://personal.benujcich.georgetown.domains,cb0qEdoAAAAJ
 Benjamin Van Durme,Johns Hopkins University,http://www.cs.jhu.edu/~vandurme,wIujAJoAAAAJ
 Benjamin W. Fuller,University of Connecticut,http://benjamin-fuller.uconn.edu,7wXWIdUAAAAJ
-Benjamin W. Wah,Chinese University of Hong Kong,http://www.cuhk.edu.hk/governance/officers/benjamin-wah/english/biography.html,WL8igVEAAAAJ
-Benjamin Wan-Sang Wah,Chinese University of Hong Kong,http://www.cuhk.edu.hk/governance/officers/benjamin-wah/english/biography.html,WL8igVEAAAAJ
+Benjamin W. Wah,Chinese University of Hong Kong,https://wah.cse.cuhk.edu.hk,WL8igVEAAAAJ

```

Comment:
Spurious newline.

Reply:
@emeryberger Thanks for reviewing the request! I could not identify the spurious newlines on Linux. I could see that the changed lines all terminate with CRLF from the output of "git diff". Do you suggest to replace all "CRLF" to "LF" in the csv files, e.g., using command like "dos2unix"? This would change almost all the lines.

Reply:
I'll merge and fix.

Reply:
Thank you for the efforts!